### ISSUE TITLE
Days are wrong

### ISSUE THREAD

**Issue by LucaGabi** [achreffaidi/customizable_date_picker#1]
![image](https://user-images.githubusercontent.com/37976898/97772998-cae2dc00-1b54-11eb-8b3b-c2235033deea.png)

**Comment by achreffaidi** [comment]
Yeah apparently there's a problem 😕 ... i'll try to fix it. Thanks for notifying me @LucaGabi .

**Comment by achreffaidi** [comment]
![Screenshot 2020-10-31 at 5 43 04 PM](https://user-images.githubusercontent.com/32805130/97784679-8b41e180-1ba0-11eb-8513-f34e494817b9.png)

The example works perfectly for me! 
Did it happen only once for you or you still having the same error ?

**Comment by LucaGabi** [comment]
I has to do with daylight hour change .. 

10x for the plugin!
 
:)

**Comment by achreffaidi** [comment]
That makes sense! that's why it doesn't appear for me, it's because we don't have daylight hour change in my country. then i thing changing the default time from midnight to another hour should solve the problem.
Thanks @LucaGabi ^^ 
i was planning to refactor the project and improve the code quality but i couldn't find time to ... the code looks a bit messy. 
I'll try to fix the bug and update the plugin during this week.
